presentation/src/commonMain/kotlin/com/peto/droidmorning/presentation/login/vm/AuthUiEvent.kt

```
@@ -0,0 +1,4 @@
+package com.peto.droidmorning.presentation.login.vm
+
+class AuthUiEvent {
+}
```

detekt (1.23.8)

[warning] The class or object AuthUiEvent is empty.

(detekt.empty-blocks.EmptyClassBlock)